### PR TITLE
Monomorphic Types

### LINKED ISSUE #2: Monomorphic Lenses?
The other Swift Lens framework, `Monocle`, has taken to a data-accessor-esque implementation of Lenses, and I think there's something to the idea of including a record-accessor structure with Focus.  Especially since Swift doesn't allow polymorphic type aliases, and Monomorphic lenses are just _easier_ to reason about for the general Swift programmer.

### PR DESCRIPTION
**Pull request by CodaFi**
Fixes #2

### CODE REVIEW

File: Focus/Monomorphic.swift [hunk 1]
```diff
@@ -0,0 +1,112 @@
+//
+//  Monomorphic.swift
+//  Focus
+//
+//  Created by Robert Widmann on 8/23/15.
+//  Copyright © 2015 SypeLift. All rights reserved.
+//
+
+public struct MonomorphicLens<S, A> : LensType {
```
Comment (CodaFi) [inline Focus/Monomorphic.swift]: Got better names for all of these @pthariensflame?

File: Focus/Monomorphic.swift [hunk 2]
```diff
@@ -0,0 +1,112 @@
+//
+//  Monomorphic.swift
+//  Focus
+//
+//  Created by Robert Widmann on 8/23/15.
+//  Copyright © 2015 SypeLift. All rights reserved.
```
Comment (pthariensflame) [inline Focus/Monomorphic.swift]: This is a rather strange typo.  :P

File: Focus/Monomorphic.swift (hunk 1 above)
Comment (pthariensflame) [inline Focus/Monomorphic.swift]: How about `Simple(Lens|Iso|Prism)`?  That would be in line with the original `lens` library's conventions.

Comment (CodaFi) [inline Focus/Monomorphic.swift]: I like that.  Will update

**Comment by CodaFi** [comment]
There.  Should be ready now.

**Comment by pthariensflame** [comment]
Yay!  How about some tests?

**Comment by CodaFi** [comment]
Done!

**Comment by pthariensflame** [comment]
:+1:

**Comment by CodaFi** [comment]
:boat:

**Pull request merged**